lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -47,6 +48,32 @@ function extractContentTypeAndValue(content: Content): [DialogType, string] {
   }
 }
 
+type ContentSelectorButtonProps = ComponentProps<typeof ButtonBase> & {
+  contentType?: string;
+};
+
+function ContentSelectorButton({
+  children,
+  contentType = 'pdf',
+  ...rest
+}: ContentSelectorButtonProps) {
+  return (
+    <ButtonBase
```

Comment:
Using `ButtonBase` for now. To be replaced with unstyled `Button` once we support it.